src/share/utf8/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(CMAKE_REQUIRED_LIBRARIES iconv)
+check_symbol_exists(iconv "iconv.h" HAVE_ICONV_LIB)
+
+add_library(utf8 STATIC
+    charset.c
+    iconvert.c
+    utf8.c)
+
+target_link_libraries(utf8 PUBLIC grabbag $<$<BOOL:${HAVE_ICONV_LIB}>:iconv>)
```

Comment:
```suggestion
target_link_libraries(utf8 PUBLIC grabbag $<$<BOOL:${WIN32}>:win_utf8_io> $<$<BOOL:${HAVE_ICONV_LIB}>:iconv>)
```

Reply:
I've checked utf8, it doesn't depend on win_utf8_io